src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header {
+  margin: 10px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+header > a {
+  width: 30%;
+  text-align: center;
+}
```

Comment:
You are styling an `a` tag directly here, which is not recommended according to the checklist. It would be better to add a class to the `a` tag and style that class instead.